lib/identity_job_log_subscriber.rb

```diff
@@ -118,7 +118,13 @@ def logger
   end
 
   def self.worker_logger
-    @worker_logger ||= ActiveSupport::Logger.new(Rails.root.join('log', 'workers.log'))
+    return @worker_logger if defined?(@worker_logger)
```

Comment:
```suggestion
@worker_logger ||= FeatureManagement.build_logger_or_stdout('log', 'workers.log')
```

Reply:
I don't feel super strongly on it I guess, I don't mind the existing implementations 🙂